sdk/monitor/azure-monitor-query/swagger/README.md

```diff
@@ -22,6 +22,7 @@ required-fields-as-ctor-args: true
 model-override-setter-from-superclass: true
 credential-types: tokencredential
 client-side-validations: true
+artifact-id: azure-monitor-query
```

Comment:
This looks like a new addition to the autorest. How were the version and SDK being loaded until this was done. I don't think I did anything special for ACR but I do have the usual builder plumbing for it.

Reply:
Looks like ACR is missing `azure-containers-containerregistry.properties` file. Adding the above to your autorest config should generate this file and also update the builder to read from this file.  This was manually done in the past, but now we have support for this in autorest.

Reply:
Thanks. I will create a TODO for myself :)